src/main/java/io/openliberty/tools/common/plugins/util/InstallFeatureUtil.java

```diff
@@ -607,13 +605,17 @@ public void installFeatures(boolean isAcceptLicense, List<String> featuresList)
                 mapBasedInstallKernel.put("license.accept", acceptLicenseMapValue);
                 mapBasedInstallKernel.put("action.install", esaFile);
                 String ext = artifactsToExt.get(esaFile);
-                if (to != null) {
-                    mapBasedInstallKernel.put("to.extension", to);
-                    debug("Installing to extension: " + to);
+                
+                if(ext!= null && ext != "" && to != null) {
+                	warn("The \"to\" parameter was specified in both build file and server.xml file.");
+                	warn("To extension from the server.xml file will be used.");
```

Comment:
This wording is confusing. I think you are saying the extension specified in the server.xml overrides the `to` extension specified in the build file. So I would suggest the following:
"The product extension location "+ext+" specified in the server.xml file overrides the `to` extension "+to+" specified in the build file."